app/templates/components/user-count.hbs

```diff
@@ -1,14 +1,7 @@
-{{#filter-nav usersList=getUser}}
-{{/filter-nav}}
-
+{{#filter-by usersList=getUser}}{{/filter-by}}
 
 <div class="userWrapper">
-
   {{#each getUser as |user|}}
-
-  {{#user-card user=user}}
-  {{/user-card}}
-
+    {{#user-card user=user}}{{/user-card}}
```

Comment:
Use angle brackets in the hbs file.

Reply:
I have fixed this change in previous PR.